config/model/yaak/control_transformer/raw.yaml

```diff
@@ -154,6 +163,18 @@ episode_builder:
         num_embeddings: ${model.episode_builder.tokenizers.continuous.steering_angle.bins}
         embedding_dim: ${embedding_dim}
 
+    points:
+      waypoints:
+        _target_: rmind.components.nn.Sequential
+        _args_:
+          - _target_: torchvision.transforms.v2.ToDtype
+            dtype:
+              _target_: hydra.utils.get_object
+              path: torch.float32
```

Comment:
if you bump to `rbyte>=0.23.0` this can probably be done during sample processing? (via duckdb sql)

Reply:
aight, then bump to latest rbyte in this pr

Reply:
it should be an equivalent of polars `.cast(polars.List(polars.List(polars.Array(polars.Float32,2))))`. Find it too complicated for sql query. Moved castings to transforms tho

Reply:
afaict this works (post groupby):
```SQL
SELECT CAST(`waypoints/waypoints_normalized` AS array<array<array<float4>>>) FROM self
```

Reply:
![image](https://github.com/user-attachments/assets/bfb2902a-c092-4e5e-acff-3f85978235fe)
seems like dataloader messes things up